user/build.xml

```diff
@@ -295,33 +271,11 @@
   <!--
       Compiles the test code for this project
   -->
-  <target name="compile.tests"
-          depends="-compile.tests.java8,-compile.tests.java11"
-          unless="compile.tests.complete">
-  </target>
-
-  <target name="-compile.tests.java8" depends="compile.dev.tests, compile.emma.if.enabled">
+  <target name="compile.tests" depends="compile.dev.tests, compile.emma.if.enabled">
     <gwt.javac srcdir="test" destdir="${javac.junit.out}"
-               excludes="com/google/gwt/langtest/**,**/super/**,**/java9/**,**/*Java9*.java,**/java10/**,**/*Java10*.java,**/java11/**,**/*Java11*.java"
+               excludes="com/google/gwt/langtest/**,**/super/**"
                processorpath="test.extraclasspath"
-               errorprone.args="
-                 -Xep:SelfAssignment:OFF
-                 -Xep:MissingCasesInEnumSwitch:OFF
-                 -Xep:SelfComparison:OFF
-                 -Xep:SelfEquals:OFF
-                 -Xep:FallThrough:OFF
-                 -Xep:ReturnValueIgnored:OFF
-                 -Xep:EqualsIncompatibleType:OFF
-                 -Xep:IdentityBinaryExpression:OFF
-                 -Xep:LoopConditionChecker:OFF
-                 -Xep:JUnitAssertSameCheck:OFF
-                 -Xep:CollectionIncompatibleType:OFF
-                 -Xep:DeadThread:OFF
-                 -Xep:ComplexBooleanConstant:OFF
-                 -Xep:ComparableType:OFF
-                 -Xep:DoNotCall:OFF
-                 -Xep:BadAnnotationImplementation:OFF
-               ">
+               release="11">
```

Comment:
Consider adding `-XDstringConcat=inline` here instead of `-noverify` in a few different places, since that generates correct bytecode until either the fix is backported to 11, or we set our minimum java level to 23+. See https://github.com/openjdk/jdk/pull/17483